data_steward/bq_utils.py

```diff
@@ -124,6 +125,42 @@ def get_validation_results_dataset_id():
     return dataset_id
 
 
+def is_validation_dataset_id(dataset_id):
+    """
+    Check if  bq_consts.VALIDATION_PREFIX is in the dataset_id
+    :param dataset_id: 
+    :return: a bool indicating whether dataset is a validation_dataset
+    """
+    compiled_exp = re.compile(bq_consts.VALIDATION_DATASET_REGEX)
+    return compiled_exp.match(dataset_id)
+
+
+def get_latest_validation_dataset_id(project_id):
+    """
+    Get the latest validation_dataset_id based on most recent creationTime. 
+    :param project_id: 
+    :return: the most recent validatioN_dataset_id
+    """
+
+    pattern = re.compile(bq_consts.VALIDATION_DATASET_REGEX)
```

Comment:
Hey @ChaoPang, I was referring to these. We do not want to add new functions to `bq_utils.py`. Any new bq function has to use `utils/bq.py`

Reply:
I see, I will move those to `utils/bq.py`

Reply:
Thanks for adding these in `utils/bq.py`, @ChaoPang. These can be removed now.